### ISSUE TITLE
Outlook 2016 emails and calendar invites are overly verbose when opened

### ISSUE THREAD

**Issue by wallaceid** [nvaccess/nvda#6373]
When opening a mail or a calendar invite, NVDA appears to read all the fields without stopping. It's very verbose, is there a way to have it say less?

**Comment by sucharugupta** [comment]
I also experience the same

**Comment by Adriani90** [comment]
@wallaceid, @sucharugupta are you still reproducing this issue? For me it works very well with NVDA 2019.1 Beta 1. It is not too verbose.

**Comment by Adriani90** [comment]
No updates on this since years, I cannot reproduce it with NvDA 2020.1. Closing as abandoned and works for me.
